cmd/root.go

```diff
@@ -0,0 +1,61 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var cfgFile string
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:   "crda",
+	Short: "Cli tool to interact with CRDA Platform.",
+	Long:  `Cli tool to interact with CRDA Platform. Perfoms synk token validation.`,
+	Args:  cobra.ExactValidArgs(1),
+}
+
+// Execute adds all child commands to the root command and sets flags appropriately.
+// This is called by main.main(). It only needs to happen once to the rootCmd.
+func Execute() {
+	if err := rootCmd.Execute(); err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+}
+
+func init() {
+	cobra.OnInitialize(initConfig)
+
+	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $PROJECT_ROOT/config.yaml)")
+	rootCmd.PersistentFlags().BoolP("debug", "d", false, "set log level to debug")
```

Comment:
PersistentFlags is used if you want to pass your flag values to subcommands. 
It sets the default value to false, when the user passes `--debug` it will be true.

Is that what you mean to ask..?